frontend/pages/media/podcast.js

```diff
@@ -78,19 +77,21 @@ const Podcast = () => {
     const filteredContent = content.filter(
       (picture) =>
         selectedCategory === "All" ||
-        picture.category.includes(selectedCategory)
+        picture.fields.category.includes(selectedCategory)
     );
     searchPodcasts(filteredContent, searchTerm);
   };
 
   // filter category content by search filter in heading, subheading or author
   const searchPodcasts = (filteredContent, searchTerm) => {
     const searchResults = filteredContent.filter((episode) => {
+      const { title, date, description } = episode.fields;
```

Comment:
Just curious, why is `{ title, date, description }` defined twice in lines 88 and 241?